client/src/components/ReviewAnswers.js

```diff
@@ -19,31 +96,56 @@ const ReviewAnswers = ({
   numOfQuestions,
 }) => {
   return (
-    <div>
+    <Container>
       <p>
         Your score is: {score} / {numOfQuestions}
       </p>
-      {questions.map(question => (
-        <Card key={question._id} className="mb-4">
-          <Card.Body>
-            <Card.Title>{question.text}</Card.Title>
-            <ul className="mb-0">
-              {question.answers.map(answer => (
-                <li
-                  key={answer._id}
-                  className={getAnswersClass(question, userAnswers, answer)}
-                >
-                  {answer.text}
-                </li>
-              ))}
-            </ul>
-          </Card.Body>
-        </Card>
-      ))}
-      <Button className="mb-4" variant="primary" href={quizName}>
-        Try again
-      </Button>
-    </div>
+      <Form>
+        <fieldset disabled>
+          {questions.map(question => (
+            <Card key={question._id} className="mb-4">
+              <Card.Header as="h5">{question.text}</Card.Header>
+              <Card.Body className="bg-white">
+                <Form.Group className="bg-light" key={question._id}>
+                  <ul className="list-unstyled">
+                    {question.answers.map(answer => {
+                      const selectedAnswers = userAnswers[question._id];
+                      return (
+                        <li
+                          style={{
+                            borderBottom: '1px solid white',
+                            position: 'relative',
+                          }}
+                          key={answer._id}
+                          className={getAnswersClass(
+                            question,
+                            userAnswers,
+                            answer,
+                          )}
+                        >
+                          {getAnswersMark(
+                            question,
+                            userAnswers,
+                            answer,
+                            selectedAnswers,
+                          )}
+                          <label className="review-answers px-2">
```

Comment:
Labels are used to label a form input. Since this is not referring to a form input, it should not be used here